app/controllers/api/v1/messages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::MessagesController < ApplicationController
+  def index
+    @greet = Message.order('RANDOM()').first
```

Comment:
API with action for greeting is properly added 🙌

Reply:
ok 👍🏽